<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255)</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>16</width>
     <height>721</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(238, 238, 238)</string>
   </property>
   <property name="text">
    <string>시간표</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(238, 238, 238)</string>
   </property>
   <property name="text">
    <string>장바구니 담기</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>811</width>
     <height>301</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::CopyAction</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string>대학</string>
    </property>
    <property name="background">
     <color>
      <red>242</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>개설학과</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>학년</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>이수구분</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>과목명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>학점-시간</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>담당교수</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>강의시간</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>강의유형</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>811</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>대학</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>대학(전체)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>소프트웨어대학</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>학과</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>소프트웨어학부</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>알고리즘</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>컴퓨터구조</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>운영체제</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>캡스톤디자인</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>수치해석</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>과목명</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>요일/교시</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>월요일</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>화요일</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>수요일</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>목요일</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>금요일</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>토요일</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>일요일</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0교시 [0800-0850]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1교시 [0900-0950]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2교시 [1000-1050]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3교시 [1100-1150]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4교시 [1200-1250]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5교시 [1300-1350]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6교시 [1400-1450]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7교시 [1500-1550]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8교시 [1600-1650]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9교시 [1700-1750]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10교시 [1800-1850]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11교시 [1900-1950]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12교시 [2000-2050]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13교시 [2100-2150]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14교시 [2200-2250]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15교시 [2300-2350]</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>캠퍼스</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>서울</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>다빈치</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(238, 238, 238)</string>
   </property>
   <property name="text">
    <string>마법사</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
